content-cache/docs/changelog.md

```diff
@@ -1,21 +1,13 @@
 # Changelog
 
-## 2024-11-28
-
-### Added
-
-- Support for TLS certificate integration. Integrating with charms that provides tls-certificates will enable HTTPS for the content cache.
-
-## 2024-10-17
-
-### Added
-
-- Support for fail-timeout, backends-path, proxy-cache-valid configuration options from integration with Content Cache Backends Config charm.
-
-## 2024-10-07
-
-### Added
-
-- Changelog for tracking user-relevant changes.
-- Basic content cache functionality with nginx.
-- Support for hostname, path, backends, protocol configuration options from integration with Content Cache Backends Config charm.
+* 2024-11-28
+  * **Added**
+    * Support for TLS certificate integration. Integrating with charms that provides tls-certificates will enable HTTPS for the content cache.
+* 2024-10-17
+  * **Added**
+    * Support for fail-timeout, backends-path, proxy-cache-valid configuration options from integration with Content Cache Backends Config charm.
+* 2024-10-07
+  * **Added**
+    * Changelog for tracking user-relevant changes.
+    * Basic content cache functionality with nginx.
+    * Support for hostname, path, backends, protocol configuration options from integration with Content Cache Backends Config charm.
```

Comment:
The formatting doesn't look right?

Reply:
I moved it to list-based from heading-based due to some issue with review dog linting due to empty section after heading.

Let me try again, and see if operator-workflow still is using that linting.

Reply:
You can also just add a short sentence or something